.pre-commit-config.yaml

```diff
@@ -7,3 +7,9 @@ repos:
     - id: check-yaml
     - id: debug-statements
     - id: check-ast
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.7.2
+    hooks:
+    - id: pyupgrade
+      args: ['--py37-plus']
+      exclude: (^_version/|^versioneer/)
```

Comment:
The ignore pattern doesn't seem to work.

src: https://github.com/psf/black/issues/395